azurerm/internal/services/containers/kubernetes_nodepool.go

```diff
@@ -121,6 +121,16 @@ func SchemaDefaultNodePool() *schema.Schema {
 					ValidateFunc: validation.IntAtLeast(1),
 				},
 
+				"os_disk_type": {
+					Type:     schema.TypeString,
+					Optional: true,
```

Comment:
presumably this defaults to `Managed` for existing disks, I guess?

Reply:
Yeah, it defaults to managed, should I add a default here even though it's optional?

Reply:
As discussed, leaving empty.